src/category_theory/shift.lean

```diff
@@ -220,6 +220,19 @@ section add_group
 variables (C) {A} [add_group A] [has_shift C A]
 variables (X Y : C) (f : X ⟶ Y)
 
+instance (i : A) : is_equivalence (shift_functor C i) :=
+begin
+  change is_equivalence (add_neg_equiv (shift_monoidal_functor C A) i).functor,
+  apply_instance,
+end
+
+@[simp] lemma shift_functor_inv_obj (i : A) (X) :
+  (shift_functor C i).inv.obj X = (shift_functor C (-i)).obj X :=
```

Comment:
Can we instead have a single lemma `(shift_functor C i).inv = shift_functor C (-i)` which gives both of these?